app/assets/stylesheets/application.scss

```diff
@@ -643,3 +643,7 @@ td.keys {
 .settings-sha {
   margin: 35px 0 10px;
 }
+
+.emoji{
+  height: 1em;
+}
```

Comment:
just a small change but I could we set a width here, I think if we do that the browser will assign the correct space for the emoji during page load i.e. stop a little page shift

Reply:
Good call, just added that 👍